### PR TITLE
Create empty comments page and add images

### PR DESCRIPTION
**Pull request by codyjrivera**
In this pull request, I add a comments page as well as some images. The comments page is currently blank -- my comments code will be added to the branch week3-changes.

I also add a link to my portfolio in README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,6 @@ This repository contains starter projects completed as part of the STEP
 internship program. Right now, this includes a personal portfolio built 
 for Google Cloud's app engine -- build and deployment instructions are
 contained in `portfolio/`.
+
+The portfolio is deployed here: 
+[https://codyjrivera-step-2020.uc.r.appspot.com](https://codyjrivera-step-2020.uc.r.appspot.com)
```
Comment (caroqliu) [inline README.md:11]: optional: If you want to link the URL as is in the markdown I don't _think_ you need to hyperlink it.

File: portfolio/src/main/webapp/src/act.html [hunk 2]
```diff
@@ -10,9 +10,18 @@
     <div id="header"></div>
     <div id="contents">
       <h3>Competitive Programming</h3>
+      <div class="image-container">
+        <img class="image"
+        alt="Programming competition participants posing for a group photo"
```
Comment (caroqliu) [inline portfolio/src/main/webapp/src/act.html:15]: Yay for `alt` tags!!!

File: README.md (hunk 1 above)
Comment (codyjrivera) [reply README.md:11]: I will remove it.

File: portfolio/src/main/webapp/src/act.html (hunk 2 above)
Comment (codyjrivera) [reply portfolio/src/main/webapp/src/act.html:15]: 👍

File: portfolio/src/main/webapp/src/com.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// File Modified by Cody Rivera June 2020
+
+/**
```
Comment (samouri) [inline portfolio/src/main/webapp/src/com.js:17]: nit: `com` is most often used to shorthand "communications", in this case I'd prefer writing out the full word i.e. `comments.js`

Comment (codyjrivera) [reply portfolio/src/main/webapp/src/com.js:17]: To be fair, this page does need to communicate more extensively with the server.

Nevertheless, I will change it in one of my next few pull requests.

**Pull request merged**